### PR TITLE
New package: TabbyML.Tabby version 0.13.1

### PR DESCRIPTION
**Pull request by ptsouchlos**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162335)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\llama-server.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\tabby.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 886.)

**Comment by DeveloperPaul123** [comment]
Tabby runs `llama-server` and `llama-server` depends on CUDA runtime being available. How could I ensure that those are installed properly?

File: manifests/t/TabbyML/Tabby/0.13.1/TabbyML.Tabby.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: dist/tabby_x86_64-windows-msvc-cuda122/llama-server.exe
+  PortableCommandAlias: llama-server
+- RelativeFilePath: dist/tabby_x86_64-windows-msvc-cuda122/tabby.exe
+  PortableCommandAlias: tabby
+UpgradeBehavior: install
+ReleaseDate: 2024-07-10
+Installers:
```
Comment (stephengillie) [inline manifests/t/TabbyML/Tabby/0.13.1/TabbyML.Tabby.installer.yaml:15]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Nvidia.CUDA
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @DeveloperPaul123,

It sounds like this package has a `Dependency`. This would install the `NVIDIA CUDA Toolkit` before installing this package.

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 886.)

**Comment by stephengillie** [comment]
Manual Validation stalled after installing dependencies:
image

**Comment by stephengillie** [comment]
I tried again, and when it paused, I opened Task Manager. `MsMpEng.exe` was taking up ~11-13% (aka 1 CPU core of an 8-core system), so I waited. Eventually, it had completed its unschedled scan and the install process proceeded. The sandbox doesn't support Defender so wouldn't have this wait. (It's one reason I use VMs instead of sandboxes for validations.)

**Comment by stephengillie** [comment]
Verify:
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**